armi/__init__.py

```diff
@@ -174,19 +173,32 @@ def cleanTempDirs(olderThanDays=None):
         time.
     """
     disconnectAllHdfDBs()
-
-    if os.path.exists(FAST_PATH):
+    if armi.context.CURRENT_MODE == armi.context.Mode.INTERACTIVE:
+        # assume no temporary directory was needed in interactive mode.
+        # This can help prevent surprises e.g. when a interactive task is running
+        # and is cancelled by the user but they just want to adjust something and
+        # rerun without losing all their working files.
+        return
+    if armi.context.APP_DATA not in armi.context.FAST_PATH:
```

Comment:
This would also simplify the logic above. No need to know about INTERACTIVE here, since we will have already mapped the Mode to the temp-ness of the fast path.

Reply:
Good idea!